outlines/generate/__init__.py

```
@@ -60,18 +64,28 @@ def __post_init__(self):
         if self.output_type is None:
             self.logits_processor = None
         else:
-            regex_string = self.output_type.to_regex()
-            self.logits_processor = RegexLogitsProcessor(
-                regex_string, self.model.tokenizer
-            )
+            if isinstance(self.output_type, CFG):
+                if isinstance(self.model, LlamaCpp):
```

We should raise this exception from the `LlamaCpp` class